ONNX_HUB_MANIFEST.json

```diff
@@ -3014,9 +3856,64 @@
                 ],
                 "outputs": [
                     {
-                        "name": "resnetv17_dense0_fwd",
+                        "name": "resnetv16_dense0_fwd",
+                        "shape": [
+                            "N",
+                            1000
+                        ],
+                        "type": "tensor(float)"
+                    }
+                ]
+            },
+            "extra_ports": {
+                "features": [
+                    {
+                        "name": "resnetv16_pool1_fwd",
                         "shape": [
+                            0,
```

Comment:
I think the dimension here becomes 0 because not it uses symbolic dimension N as dim_param and so its dim_value will be 0... We should update it [here](https://github.com/onnx/models/blob/87d452a218093f6a60ceb62712ffe1186dce6d64/workflow_scripts/generate_onnx_hub_manifest.py#L144). Will fix it in another PR. @mhamilton723 Please let me know if the shapes here will influence the usage of onnx.hub (although I think it shouldn't). Thanks!